docs/explanations/flyscanning.md

```diff
@@ -0,0 +1,86 @@
+# Flyscanning
+
+We can classify data collections across a spatial or parameterised domain into two broad categories: step scanning, and flyscanning.
+
+* Step scanning: The system performs a sequence of discrete operations: move to the next position, settle, acquire data, repeat.
+This ensures well-defined data points but may result in longer total acquisition times.
+Bluesky plans specify this type of measurement through devices' [Movable](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.Movable),
+[Triggerable](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.Triggerable) and [Collectable](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.Collectable) protocols.
+
+* Fly scanning: Short for "on-the-fly scanning," this method involves continuous motion during data acquisition. The motion and data collection processes
+happen simultaneously, enabling faster scans and smoother coverage of the scanned domain. Devices capable of this mode implement the [Flyable](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.Flyable) and [Collectable](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.Collectable) protocols.
+
+Filewriting detectors additionally implement [WritesStreamAssets](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.WritesStreamAssets) to report the data written to disk during the scan, either at each data point in a step scan, or periodically during a flyscan.
+
+## Scan frames and collection windows
+
+Consider some trajectory in 2D space:
+
+![A flyscan trajectory with collection windows containing frames](../images/flyscan_collection_windows_and_frames.svg)
+
+Along the flyscan trajectory we define discrete collection windows, which are intervals where a coherent block of data is acquired.
+The data collection block is the scan frame, defined as [a vector of lower, mid and upper points in scan space](https://blueskyproject.io/scanspec/main/explanations/technical-terms.html#frame). Note that a step scan could also be understood as a flyscan where the frames' lower and upper bounds are the same as their midpoints.
+Scan frames map directly to Bluesky [Events](https://blueskyproject.io/bluesky/main/documents.html#event), and typically corresponds to a single detector exposure, though they may be composed of multiple detector frames.
+
+## Synchronisation mechanisms
+
+A flyscan must define the start of each collection window and the acquisition of scan frames within those. Various mechanisms can be used, depending on the hardware and experimental requirements.
+
+### Collection window triggers
+
+Collection windows are level-triggered, meaning the window is open while the level is high.
```

Comment:
I don't think this sentence is true, they're not always level triggered, sometimes they're not triggered at all like in the software case. Maybe instead of talking about triggers we should talk about "Collection window definitions" instead?

Reply:
Deleted. Though I would argue that the software case is akin to a level-trigger, as in, the window is open between these two software invocations